js/coinex.js

```diff
@@ -1536,22 +1641,195 @@ module.exports = class coinex extends Exchange {
 
     async fetchOrdersByStatus (status, symbol = undefined, since = undefined, limit = undefined, params = {}) {
         await this.loadMarkets ();
-        if (limit === undefined) {
-            limit = 100;
-        }
+        limit = (limit === undefined) ? 100 : limit;
         const request = {
-            'page': 1,
             'limit': limit,
+            // 'page': 1, // SPOT
+            // 'offset': 0, // SWAP
+            // 'side': 0, // SWAP, 0: All, 1: Sell, 2: Buy
         };
         let market = undefined;
         if (symbol !== undefined) {
             market = this.market (symbol);
             request['market'] = market['id'];
         }
-        const method = 'privateGetOrder' + this.capitalize (status);
+        const swap = market['swap'];
```

Comment:
We must use `handleMarketTypeAndParams` in this method, cause `market` may be undefined on this line.